packages/docs-site/src/content/docs/docs/guides/monorepos.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Monorepo guide
+---
+
+import { FileTree } from "@astrojs/starlight/components";
+
+While dmno works great in a traditional single-purpose repo, it was designed from the beginning to handle the challenges of working in a monorepo. No other config tool lets you easily share and reuse config across your monorepo without a ton of brittle custom glue code.
+
+## Monorepo structure
+
+While in a monolithic repo, you will have a single `.dmno` folder that holds all of your dmno config, in a monorepo, you'll have a single `.dmno` folder at the root in your _workspace root service_, and then each child project/service will have its own `.dmno` folder too. Here is an example:
+
+<FileTree>
+    - / (root of your project)
+      - .dmno (your root service config)
+        - **config.mts**
+      - packages
+        - my-package
+          - .dmno 
+            - **config.mts** (config for my-package service)
+        - another-package
+          - .dmno
+            - **config.mts** (config for another-package service)
+      - package.json
+</ FileTree>
+
+If you're starting fresh with `dmno`, then the [`dmno init`](/docs/reference/cli/init/) command will detect all your services and help initialize dmno in each.
+
+## Detecting child services
+
+By default, DMNO will rely on your existing tooling to detect where potential child projects are found. This is usually an array of paths or glob patterns. We look in the following locations:
+
+| Package manager / tool | Globs location |
+|---|---|
+| npm, yarn, bun | `package.json`<br/>└ `workspaces` |
+| pnpm | `pnpm-workspace.yaml`<br/>└ `packages` |
+| moonrepo | `.moon/workspace.yml`<br/>└ `projects` |
+
+In some situations, like in a large polyglot repo, or a large repo that has multiple smaller monorepos within it, you may need an alternate way of defining where to look for dmno services. In this case, you can create a `workspace.yaml` in your workspace root's `.dmno` folder. If this file is found, it will override everything else.
```

Comment:
```suggestion
In some situations, like in a large polyglot repo, or a large repo that has multiple smaller monorepos within it, you may need an alternate way of defining where to look for `dmno` services. In this case, you can create a `workspace.yaml` in your workspace root's `.dmno` folder. If this file is found, it will override everything else.
```